### PR TITLE
issue-58: Add closeMenu function and RootCloseWrapper

### PR DESCRIPTION
**Pull request by brandon-bogan**
FacetSearchBar will now close its suggestion menu when a user clicks somewhere else

### CODE REVIEW

File: src/components/FacetSearchBar.js [hunk 1]
```diff
@@ -236,7 +243,7 @@ class FacetSearchBar extends React.Component<FacetSearchBarDefaultProps, FacetSe
    */
   doSearch() {
     const callback = this.handleSearchResults;
-    this.doConfiguredSearch(`${this.state.facetValue}*`, this.props.maxValues * 2, callback, this.context.searcher);
+    this.doConfiguredSearch(`${this.state.facetValue}*`, this.props.maxValues * 2, callback, this.context.searcher); // `
```
Comment (luigivaldez) [inline src/components/FacetSearchBar.js]: What's up with this comment? Can you remove it?

**Review by luigivaldez — CHANGES_REQUESTED** [review]
See my comment about your comment.... Otherwise, this looks good.

File: src/components/FacetSearchBar.js (hunk 1 above)
Comment (klk1010) [inline src/components/FacetSearchBar.js]:
```suggestion
    this.doConfiguredSearch(`${this.state.facetValue}*`, this.props.maxValues * 2, callback, this.context.searcher);
```

Comment (brandon-bogan) [reply src/components/FacetSearchBar.js]: Fixed in latest commit

**Review by luigivaldez — APPROVED** [review]

**Pull request merged**